pkg/controller/README.md

```diff
@@ -9,11 +9,17 @@ Step 1 and Step 2 are required only when you make changes locally
 1. Run `kubectl apply -k deploy/`.
 
 ### Run outside the cluster
+1. Set env $WATCH_NAMESPACE and $LEADER_ELECTION_NAMESPACE, default value is istio-operator
 
-1. Install [Operator SDK CLI](#https://github.com/operator-framework/operator-sdk/blob/master/doc/user/install-operator-sdk.md)
+1. From operator root directory, run `go run ./cmd/manager/*.go server `
 
-1. Then run
-```
-operator-sdk up local --namespace=istio-operator --operator-flags "server"
-```
+To use Remote debugging with IntelliJ, replace above step 2 with following:
 
+1. From $OPERATOR_ROOT/cmd/manager path run
```

Comment:
Just ./cmd/manager, $OPERATOR_ROOT is not defined.

Reply:
done